android/titanium/prebuild.js

```diff
@@ -81,13 +81,18 @@ async function generateBuildProperties() {
 		buildGitHash = 'HEAD';
 	}
 
+	let buildTimestamp = process.env.TI_SDK_BUILD_TIMESTAMP;
+	if (!buildTimestamp) {
+		buildTimestamp = moment().format('YYYY/MM/DD HH:mm');
```

Comment:
it may! We should be using the same format in the manifest.json in the SDK as we use in the Ti.buildDate property at runtime (and that value should be the same on either platform). I think that Utils.timestamp basically is what will get shoved into the iOS Ti.buildDate in Obj-c, and will be used for the babel pre-filling. And I think is set to the env var we'll use and prefer above here.

So I think this really only matters for local dev builds, but yeah we should be consistent in our formatting.

I think our unit test doesn't actually verify the format of the value so it's possible we've had iOS/Android differing here for a very long (forever?) time.